src/stages/kinesis-s3.ts

```diff
@@ -34,11 +35,13 @@ export class FirehoseToS3Stage extends DataStage {
     if (props.s3Bucket) {
       this.bucket = props.s3Bucket;
     } else if (props.s3BucketProps) {
-      const bucketProps = S3Defaults.bucketProps({
-        ...props.s3BucketProps,
-        eventBridgeEnabled: true,
-      });
-      this.bucket = new s3.Bucket(this, "Stage Bucket", bucketProps);
+      const bucketProps = overrideProps(
+        {
+          eventBridgeEnabled: true,
+        },
+        props.s3BucketProps,
```

Comment:
Don't we want `eventBridgeEnabled: true` to override the `s3BucketProps`, rather than the reverse?

I think you could simply this with just
```javascript
this.bucket = S3Factory.bucket(
  this,
  "Stage Bucket",
  {
    ...props.s3BucketProps,
    eventBridgeEnabled: true,
  },
);
```

Reply:
Thanks for catching that, much cleaner.